rocon_python_comms/src/rocon_python_comms/services.py

```diff
@@ -36,6 +36,26 @@
 ##############################################################################
 
 
+def is_valid_service(service_name):
```

Comment:
`is_valid_service` is confusing...that assumes it is available AND not bad/corrupt.

`is_available` or `service_is_available` would be less confusing.